### ISSUE TITLE
Error about auto updates

### ISSUE THREAD

**Issue by tommoyer** [Hammerspoon/hammerspoon#2274]
Today when I booted my laptop, I was presented with a dialog box stating that Hammerspoon's auto update was not configured and to consult the Sparkle documentation. This appears to be an issue that the developers need to address, not something users can do on their own. Below is the dialog box that was presented.

<img width="532" alt="Pasted Graphic" src="https://user-images.githubusercontent.com/406913/71633587-1f3ec780-2be3-11ea-80ce-c2afe860f97c.png">

**Comment by latenitefilms** [comment]
Are you running a build from the public Hammerspoon website, or a custom build?

**Comment by tommoyer** [comment]
Downloaded from the website

Tom

—
Tom Moyer
Email: tommoyer@gmail.com

**Comment by latenitefilms** [comment]
That's strange. Working fine here. Can you try re-downloading from the site?

**Comment by cmsj** [comment]
Yeah that is strange - Hammerspoon is (and has always been) code signed with the proper Apple signing mechanisms. I wonder if the download was corrupted somehow?
